alica_tests/src/test/test_alica_parallel_execution.cpp

```diff
@@ -0,0 +1,38 @@
+#include "test_alica.h"
+#include <gtest/gtest.h>
+
+namespace alica::test
+{
+
+TEST_F(SingleAgentTestFixture, runSameBehaviourInParallel)
```

Comment:
can have a test with both plans & behaviours with same names attached to same state (in parallel)